spec/helpers/session.rb

```diff
@@ -0,0 +1,12 @@
+module SessionHelpers
+
+  def sign_up(user)
```

Comment:
Unused method argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used. You can also write as `sign_up(*)` if you want the method to accept any arguments but don't care about them.